src/axom/primal/operators/evaluate_integral.hpp

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/*!
+ * \file evaluate_integral.hpp
+ *
+ * \brief Consists of methods that evaluate integrals over regions defined
+ * by Bezier curves, such as 2D area integrals and scalar/vector field line integrals
+ *
+ * Line integrals are computed with simple 1D Gaussian Quadrature using rules generated
```

Comment:
Thinking ahead, we might not want to prescribe the type of quadrature in this high level description.

```suggestion
 * Line integrals are computed with 1D quadrature rules supplied
```